src/Microsoft.AspNet.OData/Extensions/HttpConfigurationExtensions.cs

```diff
@@ -266,6 +268,39 @@ public static void AddODataQueryFilter(this HttpConfiguration configuration, IAc
             configuration.Services.Add(typeof(IFilterProvider), new QueryFilterProvider(queryFilter));
         }
 
+        /// <summary>
+        /// Enables support for case insensitive model binding.
+        /// </summary>
+        /// <param name="configuration">The server configuration.</param>
+        public static void EnableODataCaseInsensitiveModelBinding(this HttpConfiguration configuration)
```

Comment:
```suggestion
        public static void EnableCaseInsensitiveModelBinding(this HttpConfiguration configuration)
```

Reply:
We have agreed to retain `EnableODataCaseInsensitiveModelBinding`
#Resolved